### ISSUE TITLE
UIExplorer doesn't work on clean installation

### ISSUE THREAD

**Issue by edgarmariorivera** [facebook/react-native#7151]
Guys, I clone https://github.com/facebook/react-native.git then run npm install

<img width="1920" alt="captura de pantalla 2016-04-22 a las 13 15 42" src="https://cloud.githubusercontent.com/assets/3542557/14741173/562befda-088c-11e6-826a-24b91bfdb2ef.png">

Then open UIExplorer from Xcode and I got this error.

Any clues, what I'm missing here?

**Comment by ptomasroos** [comment]
I have the same problem on a existing project. Something got broken today :) Same error

**Comment by grabbou** [comment]
Works fine as of current master (reproducing issues right now). Would you mind double checking?

**Comment by christopherdro** [comment]
This was part of the reason why travis tests were failing recently.

I believe this commit resolved the issues. https://github.com/facebook/react-native/commit/6fac6db4d43dcfe5965e92fa15f3d62add8ff9fd

**Comment by edgarmariorivera** [comment]
I cloned the repo again and run npm install, after last commit build from Xcode is working. But once I tried to run the app on my iPhone I got this error

![error-running-reactive-native-iphone](https://cloud.githubusercontent.com/assets/3542557/14762765/018a277c-097b-11e6-93aa-4d97909c038f.jpg)

If I run the app on the iPhone simulator, is working.

I got the same error running TicTacToe, Movies.

2048 has a runtime error on RCTUIManager.h 
[(NSMutableDictionary<NSNumber *, UIView *> *)viewRegistry removeObjectForKey:rootReactTag];

Looks like running apps on device is not working by default

In other hands, this examples are showing errors on the simulator

Components --> TextInput --> [RCTShadowText.m:227] <Text> can't have any children except <Text>, <Image> or raw strings. This goes away pressing Esc

API --> Border --> [tid:com.facebook.react.RCTExceptionsManagerQueue] Unhandled JS Exception: Can't find variable: React. This now blocks the app, reinstall is required to run it again.

API --> Border Shadow --> [tid:com.facebook.react.RCTExceptionsManagerQueue] Unhandled JS Exception: Can't find variable: React. This now blocks the app, reinstall is required to run it again.
